lib/utils/MyLocalizations.dart

```diff
@@ -6,10 +6,17 @@ import 'package:flutter/services.dart';
 class MyLocalizations {
   Locale locale;
   static Map<dynamic, dynamic>? _localisedValues;
+  static Map<dynamic, dynamic>? _englishValues;
 
   MyLocalizations(this.locale) {
     locale = locale;
     _localisedValues = null;
+    loadEnglishTranslations();
+  }
+
+  static Future<void> loadEnglishTranslations() async {
```

Comment:
Make it a private function since it is not intended to be used outside the class.

`_loadEnglishTranslations`